build-scripts/build_compiler.js

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2018 The Closure Compiler Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const {spawn, spawnSync} = require('child_process');
+const ncp = require('ncp');
+const Semver = require('semver');
+const version = require('../lerna.json').version;
```

Comment:
Could you give a more specific name saying what this version represents exactly and what it should look like?

Reply:
well, I meant a comment saying what it should look like, but a more specific variable name would also be good.

Reply:
I deleted it.